completion.py

```diff
@@ -44,7 +47,30 @@ def get_completion(clones_dir: str, repo: str) -> tuple[float, 'TranslatorsData'
             hide_reserved=False,
             api_url='',
         ).completion
-    return completion, translators_data, branch
+
+    try:
+        # Fetch commit from before 30 days ago and checkout
+        commit = subprocess.check_output(
+            ['git', 'rev-list', '-n', '1', '--before="30 days ago"', 'HEAD'],
+            cwd=clone_path,
+            text=True,
+        ).strip()
+        subprocess.run(['git', 'checkout', commit], cwd=clone_path, check=True)
+        with TemporaryDirectory() as tmpdir2:
```

Comment:
```suggestion
        with TemporaryDirectory() as tmpdir:
```

using same variable name as earlier will work just fine